cypress/support/e2e.ts

```diff
@@ -0,0 +1,6 @@
+import { configure } from '@testing-library/cypress';
+
+import './commands';
+import './type';
+
+configure({ testIdAttribute: 'data-testid' });
```

Comment:
The team coventions is to name it:

```suggestion
configure({ testIdAttribute: 'data-test-id' });
```

As raised in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/135#discussion_r1329552737